src/Layout/PresentationMenu/PresentationSelection/PresentationSelection.tsx

```diff
@@ -32,9 +32,10 @@ export class PresentationSelection extends React.Component<PresentationSelection
         return (
             <div className="field">
                 <label className="label">
-                    <TooltipInfoRight
+                    <TooltipInfo
                         text={"Trykk på dette ikonet for å lese om oversikten."}
                         url={WIKIURL?.concat("#Visning")}
```

Comment:
Inconsistent way of handling "url" property. Why isn't it done in the same way in ComputationMenu? Needs addressing.